virttest/utils_test/libvirt.py

```diff
@@ -2022,6 +2022,7 @@ def set_vm_disk(vm, params, tmp_dir=None, test=None):
     driver_iothread = params.get("driver_iothread", "")
     mnt_path_name = params.get("mnt_path_name", "nfs-mount")
     exp_opt = params.get("export_options", "rw,no_root_squash,fsid=0")
+    exp_dir = params.get("export_dir")
```

Comment:
The exp_dir as None here will break other cases without given this and use default path.